docker/build/Dockerfile

```diff
@@ -11,7 +11,7 @@ RUN \
   apt-get clean && \
   rm --recursive --force /var/lib/apt/lists/*
 
-COPY *-requirements.txt /tmp/
+COPY *.txt /tmp/
```

Comment:
It isn't too wide ? What happens if someone create a "readme.txt" or something like this ?